src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
```

Comment:
<img width="855" alt="Screenshot 2022-01-19 at 13 08 27" src="https://user-images.githubusercontent.com/33026471/150296788-1e304f9e-1fe0-41bf-8a83-57856ee3f425.png">
from task description this method should return String and not color, let's fix that.